getting_started/introduction/key_concepts_overview.rst

```diff
@@ -0,0 +1,99 @@
+.. Intention: only to introduce only a handful of key concepts and avoid a big
+   cognitive load. Readers will then be reminded of the concepts further in the
+   getting started series, reinforcing their learning.
+
+.. _doc_key_concepts_overview:
+
+Overview of Godot's key concepts
+================================
+
+Every game engine revolves around abstractions you use to build your
+applications. In Godot, a game is a **tree** of **nodes** that you group
+together into **scenes**. You can then wire these nodes so they can communicate
+using **signals**.
+
+These are the four concepts you will learn here. We're going to look at them
+briefly to give you a sense of how the engine works. In the getting started
+series, you will get to use them in practice.
+
+Scenes
+------
+
+In Godot, you break down your game in reusable scenes. A scene like a character,
+a weapon, a menu in the user interface, a single house, an entire level, or
+anything you can think of. Godot's scenes are flexible; they fill the role of
+both prefabs and scenes in some other game engines.
+
+.. image:: img/key_concepts_main_menu.png
+
+You can also nest scenes. For example, you can put your character in a level,
+and drag and drop a scene as a child of it.
+
+.. image:: img/key_concepts_scene_example.png
+
+Nodes
+-----
+
+A scene is composed of one or more **nodes**. Nodes are your game's smallest
+building block that you arrange into trees. Here's an example of a character's
+nodes.
+
+.. image:: img/key_concepts_character_nodes.png
+
+It is made of a ``KinematicBody2D`` node named "Character", a ``Sprite2D``, a
+``Camera2D``, and a ``CollisionShape2D``.
+
+.. note:: The node names end with "2D" because this is a 2D scene. Their 3D
+          counterpart have names that end with "3D".
+
+Notice how nodes and scenes look the same in the editor. When you save a tree of
+nodes as a scene, it then acts as a single node, hiding its internal structure.
```

Comment:
This is misleading. The structure isn't really hidden, you can still get to the children in the editor by enabling "Editable Children", and you can get to the children in the code by using `get_child`/`get_node`/etc. Instanced scenes are also still very visible due to the clapperboard icon.

Reply:
But the intended usage of the design, from an Object Orientation standpoint, is as stated, so I think as an overview of how it should work / be perceived by users, this is sufficient. You can always dig into the details of how to subvert the default state in-editor/at runtime in later docs.

Reply:
I agree @willnationsdev, but we should still try to phrase things such that it's neither misleading nor confusing.

How about "it then shows up as a single node, with its internal structure hidden in the editor"?